src/priyanka_Panat/Shop.java

```diff
@@ -0,0 +1,123 @@
+package priyanka_Panat;
+
+/*
+ Assignment 7
+ */
+public class Shop{
+	int maggieQuantity;
+	int dosaQuantity;
+	int oilQuantity;
+	int panipuriQuantity;
+	int masalaQuantity;
+	
+	void setQuantity(int maggie,int dosa,int oil,int panipuri,int masala){
+		maggieQuantity=maggie;
+		dosaQuantity=dosa;
+		oilQuantity=oil;
+		panipuriQuantity=panipuri;
+		masalaQuantity=masala;
+	}
+	
+	void customerShopping(int maggieQty,int dosaQty,int oilQty,int panipuriQty,int masalaQty){
+			if(maggieQty>0){
+				if(maggieQty<=maggieQuantity){
+					int leftmaggie=maggieQuantity-maggieQty;
+					System.out.println("Sorry we have only" + maggieQuantity + " packets of maggie available ,please purchase " + leftmaggie + "packets later.");
+				}
+				else{
+					System.out.println("Maggie packets sold are"+maggieQuantity);
```

Comment:
Why we printing the OOS statements here, we have separate requirement for this FR2 and FR3